content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -155,145 +124,146 @@ enum Level {
 }
 ```
 
-This data model will be used to store incoming requests from your Worker.
+This data model will be used to fetch logs from your Worker.
 
-## 5. Create a repository and push to GitHub
+## 3. Create a repository and push to GitHub
 
 To prepare for the steps ahead, let's [create a private repository](https://github.com/new) on GitHub.
-
-![Create a private repository](../../../doc-images/guides/deploy/create-repo.png)
-
 Next, initialize your repository, then push your changes up to GitHub.
 
 ```terminal
-git remote add origin https://github.com/<username>/prisma-mongodb-cloudflare
+git remote add origin https://github.com/<username>/prisma-cloudflare-accelerate
 git add .
 git commit -m "initial commit"
 git push -u origin main
 ```
 
-You're ready to import your project into the Prisma Data Platform.
-
-## 6. Import your Project into the Prisma Data Platform
-
-With Cloudflare Workers, you can't directly access your database because there is no TCP support. Fortunately, Prisma has your back with the [Data Proxy](/data-platform/classic-projects/data-proxy).
-
-1. To get started, sign up for a free [Prisma Data Platform account](https://cloud.prisma.io/).
-
-   <Admonition type="info">
-
-   **Note**<br /><br />
-   A GitHub account is required to sign up for the Prisma Data Platform.
-
-   </Admonition>
-
-   ![Signup for Prisma Data Platform](../../../doc-images/data-platform/pdp-signup.png)
-
-2. Click **New Project** and then select **Import a Prisma repository**.
-3. Fill in the repository and project details, and click **Next**.
-
-   ![Import a Project](./images/450-03-import-project.png)
-
-4. Next, you connect the Prisma Data Platform to the MongoDB Atlas database and set up the Data Proxy.
+You're ready to create a project in [Prisma Data Platform](https://console.prisma.io).
 
-   1. Paste the MongoDB Atlas connection string which also includes the database name as described in [1. Get a MongoDB Atlas connection string](#1-get-a-mongodb-atlas-connection-string).
-   2. Under **Location**, select a Data Proxy location that is geographically close to your MongoDB database location.
-   3. Enable **Static IPs** and copy and paste them in the _Network Access_ screen in MongoDB Atlas.
+## 4. Enable Accelerate in Prisma Data Platform
 
-      ![Connect your Database](./images/450-04-connect-db.png)
+With Cloudflare Workers, you can't directly access your database because there is no TCP support. Fortunately, Prisma has your back with [Prisma Accelerate](/data-platform/accelerate).
 
-   4. Click **Create Project** to test the connection and set up the Data Proxy.
+To get started, sign up for a free [Prisma Data Platform account](https://console.prisma.io/), enable Accelerate and generate an Accelerate connection string.
```

Comment:
```suggestion
To get started:
1. Sign up for a free [Prisma Data Platform account](https://console.prisma.io/)
1. Create a project
1. Navigate to the project you created and enable Accelerate
1. Generate an Accelerate connection string
```

Feel free to play around with the wording here. Since Accelerate is GA, I think Screenshots would be helpful for visualizing the steps here.